var/spack/repos/spack_repo/builtin/packages/muparser/package.py

```diff
@@ -31,13 +35,13 @@ class Muparser(CMakePackage, Package):
     )
     variant("shared", default=True, description="enable shared libs", when="build_system=cmake")
 
-    # Non-CMake build system is not supported by windows
-    conflicts("platform=windows", when="@:2.2.5")
-    build_system(conditional("cmake", when="@2.2.6:"), "generic", default="cmake")
+    build_system(conditional("cmake", when="@2.2.6:"), "autotools", default="cmake")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
 
+
+class CMakeBuilder(CMakeBuilder):
```

Comment:
I've never been a fan of `class A(A):`, but hey... it's allowed by Python :melting_face: